src/ontology/mondo-edit.obo

```diff
@@ -170190,7 +170191,7 @@ is_a: MONDO:0021106 {source="Wikipedia:Laminopathy"} ! laminopathy
 is_a: MONDO:0976138 {source="OMIM:169500", source="https://orcid.org/0000-0001-5208-3432"} ! leukodystrophy, demyelinating, adult-onset
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
 relationship: has_characteristic MONDO:0021152 {source="OMIMPS:169500"} ! inherited
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6637 {source="MONDO:mim2gene_medgen", source="OMIM:169500"} ! LMNB1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6637 {source="MONDO:mim2gene_medgen"} ! LMNB1
```

Comment:
I confirmed this removal of the OMIM reference is correct as it does not meet our rules https://omim.org/entry/169500. I don't believe we have set a SOP yet on what to do when only the `MONDO:mim2gene_medgen` source remains on a gene association.